pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -314,6 +314,10 @@ func updateCsiConfigMapOwnerRefs(ctx context.Context, namespace string, clientse
 // CephFilesystemSubVolumeGroup) or for other supplementary entries, the clusterID should be unique
 // and different from the namespace so as not to disrupt CephCluster configurations.
 func SaveClusterConfig(clientset kubernetes.Interface, clusterID, clusterNamespace string, clusterInfo *cephclient.ClusterInfo, newCsiClusterConfigEntry *CSIClusterConfigEntry) error {
+	if EnableCSIOperator() {
```

Comment:
can we do same change in SaveCSIDriverOptions as well?

Reply:
Updated, good catch